examples/sequoia-155-dev-toolkit.hcl

```diff
@@ -0,0 +1,59 @@
+packer {
+  required_plugins {
+    macstadium-orka = {
+      version = "= 3.0.1"
+      source  = "github.com/macstadium/macstadium-orka"
+    }
+  }
+}
+
+source "macstadium-orka" "image" {
+  source_image      = "ghcr.io/macstadium/orka-images/sequoia:latest" // This image has the latest version of Orka VM tools already pre-installed 
+  image_name        = "packer-sequoia-155-dev-toolkit"
+  image_description = "MacOS Sequoia 15.5 image created with Packer!"
+  orka_endpoint     = "http://10.221.188.20"
+  orka_auth_token   = "YOUR_ORKA_SERVICE_ACCOUNT_TOKEN_HERE"
+  ssh_username      = "admin"
+  ssh_password      = "admin"
```

Comment:
Same for the SSH credentials. I would make these vars as well. The defaults can be admin/admin

Reply:
Will do, thanks!